app/src/main/res/values/static_strings.xml

```diff
@@ -53,7 +53,7 @@
     <string name="link_text_view_type_announcement" translatable="false">link</string>
 
     <!-- Name of the application for about page -->
-    <string name="daylight_app_name" translatable="false">Firefox Daylight</string>
+    <string name="daylight_app_name" translatable="false" moz:RemovedIn="103" tools:ignore="UnusedResources">Firefox Daylight</string>
```

Comment:
You can actually just remove this completely because this is a static string so it's not actually translated. Easy way to verify is to search for "daylight_app_name" and see that it doesn't appear in any of the `strings.xml`